pulsar-common/src/main/java/org/apache/pulsar/common/net/ServiceURI.java

```diff
@@ -106,7 +106,14 @@ public static ServiceURI create(String uriStr) {
 
         return create(uri);
     }
-
+    public static void validate(String urlStr) {
```

Comment:
I'm not sure if we have to enhance this method to check pulsar supported service name.
- pulsar
- pulsar+ssl
- http
- https

Reply:
I think we should

The service URL can only start with pulsar or pulsar+ssl
The web service URL can only start with http or https

Reply:
I see